angular.json

```diff
@@ -497,10 +497,6 @@
               {
                 "command": "nx build-package eslint-plugin"
               },
-              // uncomment when there are eslint-plugin migrations
```

Comment:
Revert this change

Reply:
Sounds good. Not sure why yarn format is doing that but every time formatting is run this gets removed.